optimum/exporters/onnx/base.py

```diff
@@ -319,7 +319,7 @@ def rename_ambiguous_inputs(self, inputs) -> dict[str, dict[int, str]]:
         """
         return inputs
 
-    def ordered_inputs(self, model: PreTrainedModel) -> dict[str, dict[int, str]]:
+    def ordered_inputs(self, model: "PreTrainedModel") -> dict[str, dict[int, str]]:
```

Comment:
We can add `from __future__ import annotations`. I can do that in a separate PR if that sounds good to you

Reply:
yeah ofc, let's do that in another PR.